docs/cloud_build_standards.md

```diff
@@ -0,0 +1,87 @@
+## Cloud build standards
+
+### Account Layout
+
+- Separate accounts for separate concerns i.e. CI, Ops, Prod etc. This decouples usage types into it's own account structure making it a more logical separation due to IAM being global and tagging naming possibly overlapping for common services.
+- Use cross account federation where possible for users
+- Have separate audit accounts with restricted policies on that accounts
+
+### Account naming
+
+The name of the AWS account needs to contain useful information, below is a list of things that should be contained within it
+
+| Type | Description | Usage |
+| ---- | ------ | ------ |
+| Directorate | Directorate Cost number | 1234 |
+| Cost Code | Cost code number | 3456789 |
+| Portfolio | Portfolio abbreviation | x |
+| Project Name | Name of the project | Projectx |
+| Env Name | Name of environment for account | test, notprod, prod, ops |
+
+The overall Format would be `DIRECTORATE-COSTCODE_PORTFOLIO-PROJECT-ENV`
```

Comment:
I don't like the mix of hyphens and underscores, it can be confusing to remember / type

Reply:
This only happens when you create the AWS account itself, it's to make the costs easier to see when the bill comes in as an invoice

Reply:
Ah ok, this is fine then